#summary Frequently Asked Questions
#labels Featured

==Does msatCommander work in OS X 10.3?==
 As of version 0.8.0, msatcommander requires OSX 10.4.x

==Will msatCommander work with Chromas files?==
 In > 0.4.4, Chromas files should work.

==Is msatCommander a universal binary?==
 Yes.

==Why the Beachball?==
 You will notice that (at least in the osx version of msatCommander) you get the beachball while the program is running.  This is *normal*.  

 At some point (not a priority) I will use python threads to get rid of the beachball.  I will also implement a progress bar.  For now (0.4.5 and above), a popup window indicates completion.

== What is searching performance like? ==

 _Pretty good_.  Here are some examples:

  * Scanning a single ~80,000 bp sequence from _Didelphis virgniana_ for all repeats took roughly 28 seconds on a pentium 4 @ 3 ghz.

  *  Scanning 10,000 reptile contigs (approximately 16% of the entire genome) for tetra-nucleotide repeats took roughly 2 hours on a pentium D @ 3.4 ghz.  Six-thousand, one hundred twenty-four sequences contained repeats; 25,434 repeat arrays were identified (Thanks go to Nick Crawford, Savannah River Ecology Lab, for the information.)

  * Scanning 96 contigs for all microsatellite repeats (mean contig length ~ 400bp) took roughly 33 seconds on a DP 867 mhz G4 powermac.  

  * Scanning the same 96 contigs for all microsatellite repeats from above on a Core Duo 2 Intel iMac @ 2.16 ghz took roughly 16 seconds.

  * GUI and command-line speeds are roughly the same (+/- 0.2 seconds)

== What is the success rate of designed primers? ==
  Primers designed using the design *and* tagging features of msatcommander have yielded approximately 85% amplification success.  This *does not* mean 85% of your primer will be polymorphic.  It does mean that roughly 85% should amplify some (perhaps monomorphic) product.  

  This success rate is largely due to the fact that the primer3 program (the primer design engine) is pretty darn good at what it does.

  *_Note_*:  this is dependent on your sequence quality, read-lengths, and organisms.  For example, organisms with transposable elements (gymnosperms) are finicky, to say the least.